ports/mpark-patterns/vcpkg.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "mpark-patterns",
+  "version-date": "2019-10-03",
+  "description": "MPark.Patterns is an experimental pattern matching library for C++17.",
+  "homepage": "https://github.com/mpack/patterns",
```

```suggestion
  "homepage": "https://github.com/mpark/patterns",
```
It has not been fixed.

---

this time has fixed. I only have notice the '/'